src/components/TripItem.jsx

```diff
@@ -65,13 +90,26 @@ export const TripItem = ({ geojson, trip, withDateHeader }) => {
       {withDateHeader && <ListSubheader>{day}</ListSubheader>}
       <ListItem className="u-pl-1-s u-pl-2" button onClick={handleClick}>
         <ListItemIcon>
-          <Avatar icon={pickModeIcon(mainMode)} color={modeToColor(mainMode)} />
+          <Avatar
+            icon={AvatarIcon}
+            ghost={purpose === undefined}
```

Comment:
let's  test TripItem component has Avatar changed according to purpose

Reply:
*Les tests sur les composants viendront 👍 *